app/templates/ui/app/detail/similar-directive.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  'use strict';
+
+  angular.module('ml.similar', []);
+
+}());
+
+(function () {
+
+  'use strict';
+
+  angular.module('ml.similar')
+    .directive('mlSimilar', mlSimilar);
+
+  mlSimilar.$inject = ['MLRest'];
+
+  function mlSimilar(mlRest) {
+    return {
+      restrict: 'E',
+      templateUrl: 'app/detail/similar-directive.html',
+      scope: { uri: '@' },
+      link: function(scope, iElement, iAttrs, ctrl) {
+        mlRest.extension('extsimilar', { method: 'GET', params: { 'rs:uri': iAttrs.uri, 'rs:limit': 10 }}).then(function(response) {
```

Comment:
This should be `scope.uri` (so that the value get's parsed/evaluated) instead of referencing the attribute directly. Also, could you parameterize the limit value as well?